content/en/docs/tasks/debug-application-cluster/crictl.md

```diff
@@ -0,0 +1,195 @@
+---
+reviewers:
+- Random-Liu
+- feiskyer
+- mrunalp
+title: Crictl
+---
+
+crictl provides a CLI for CRI-compatible container runtimes. This allows the CRI runtime developers to debug their runtime without needing to set up Kubernetes components.
```

Comment:
This document is for users not "runtime developers".

It helps users inspect and debug container runtime and applications on a Kubernetes node.